docs/doc_test/os-commands.rendered

```diff
@@ -0,0 +1,61 @@
+{% include navigation.html %}
+{% raw %}
+
+# OS commands
+
+It is possible to run OS commands using the following go template functions:
+
+* `exec` returns the result of a shell command as structured data.
+* `run` returns the result of a shell command as a string.
+
+## exec
+
+### Razor
+```
+First result: test2
+Second result: test3
+SomeData: test2
+SomeData2: test3
+```
```

Comment:
@jocgir The critical part of the test is here. Before this change, there was an empty line after the map was printed.